core/domain/story_jobs_one_off_test.py

```diff
@@ -251,3 +251,69 @@ def _mock_get_story_by_id(unused_story_id):
         for x in output:
             self.assertRegexpMatches(
                 x, 'Expected description to be a string, received 123')
+
+
+class PopulateStoryThumbnailSizeOneOffJobTests(test_utils.GenericTestBase):
+
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    STORY_ID = 'story_id'
+
+    def setUp(self):
+        super(PopulateStoryThumbnailSizeOneOffJobTests, self).setUp()
+
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.TOPIC_ID = topic_fetchers.get_new_topic_id()
+        self.story_id_1 = 'story_id_1'
+        self.save_new_topic(
+            self.TOPIC_ID, self.albert_id, name='Name',
+            description='Description',
+            canonical_story_ids=[self.story_id_1]
+        )
+        self.process_and_flush_pending_mapreduce_tasks()
+
+    def test_thumbnail_size_job_thumbnail_size_is_present(self):
+        self.save_new_story_with_story_contents_schema_v1(
+            self.STORY_ID, 'image.svg', '#F8BF74', 21131, self.albert_id,
+            'A title', 'A description', 'A note', self.TOPIC_ID)
+        topic_services.add_canonical_story(
+            self.albert_id, self.TOPIC_ID, self.STORY_ID)
+
+        story_model = story_models.StoryModel.get(self.STORY_ID)
+        story = story_fetchers.get_story_from_model(story_model)
+        self.assertEqual(21131, story.thumbnail_size_in_bytes)
+
+        # Start migration job.
+        job_id = (
+            story_jobs_one_off
+            .PopulateStoryThumbnailSizeOneOffJob.create_new())
+        story_jobs_one_off.PopulateStoryThumbnailSizeOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        updated_story = (
+            story_fetchers.get_story_by_id(self.STORY_ID))
+        self.assertEqual(21131, updated_story.thumbnail_size_in_bytes)
+
+    def test_thumbnail_size_job_thumbnail_size_is_not_present(self):
+        self.save_new_story_with_story_contents_schema_v1(
+            self.STORY_ID, 'image.svg', '#F8BF74', None, self.albert_id,
+            'A title', 'A description', 'A note', self.TOPIC_ID)
+        topic_services.add_canonical_story(
+            self.albert_id, self.TOPIC_ID, self.STORY_ID)
+
+        story_model = story_models.StoryModel.get(self.STORY_ID)
+        story = story_fetchers.get_story_from_model(story_model)
+        self.assertEqual(None, story.thumbnail_size_in_bytes)
+
+        # Start migration job.
+        job_id = (
+            story_jobs_one_off
+            .PopulateStoryThumbnailSizeOneOffJob.create_new())
+        story_jobs_one_off.PopulateStoryThumbnailSizeOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        updated_story = (
+            story_fetchers.get_story_by_id(self.STORY_ID))
+        self.assertEqual(None, updated_story.thumbnail_size_in_bytes)
```

Comment:
Since we are testing the job here, it would be better if we compared the results of the job with the expected results. (See other tests in [story_jobs_one_off_test.py](https://github.com/oppia/oppia/blob/6b1f47337c79a021bf7b30ebce33f8c94390903a/core/domain/story_jobs_one_off_test.py#L120))

Reply:
Done